packages/squiggle-lang/__tests__/TS/TestHelpers.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  run,
+  Distribution,
+  squiggleExpression,
+  errorValueToString,
+  errorValue,
+  result,
+} from "../../src/js/index";
+
+export function testRun(x: string): any {
+  //: result<squiggleExpression, errorValue> => {
+  return run(x, { sampleCount: 1000, xyPointLength: 100 });
+}
+
+export function failDefault() {
+  expect("be reached").toBe("codepath should never");
+}
+
+/**
+ * This appears also in `TestHelpers.res`. According to https://www.math.net/percent-error, it computes
+ * absolute error when numerical stability concerns make me not want to compute relative error.
+ * */
+export function expectErrorToBeBounded(
```

Comment:
Having four non-named arguments is not ideal.

Reply:
Isn't ``isCloseTo`` basically the same as this?

Reply:
`expect(received).toBeCloseTo(expected)` tests `abs(expected - received) < 5e-{digits}`. This is pretty terrible when checking cases across orders of magnitude. `expectErrorToBeBounded` takes into account order of magnitude. You give an `epsilon` which bounds the _percent error_.